name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/OnvistaPDFExtractor.java

```diff
@@ -0,0 +1,533 @@
+                        })
+                        
+                        .section("notation", "shares", "date")
+                        .find("Nominal (Ex-Tag )?Zahltag (.*etrag pro .*)?(Zinssatz.*)?")
+                        //STK 28,000 02.03.2015 04.03.2015 EUR 0,088340
+                        .match("(?<notation>^EUR|^STK) (?<shares>\\d{1,3}(\\.\\d{3})*(,\\d{3})?) (\\d+.\\d+.\\d{4}+) (?<date>\\d+.\\d+.\\d{4}+)(.*)")
+                        .assign((t, v) -> {
+                            String notation = v.get("notation");
+                            if (notation != null && !notation.equalsIgnoreCase("STK"))
+                            {
+                                // Prozent-Notierung, Workaround..
+                                t.setShares((asShares(v.get("shares")) / 100));
+                            }
+                            else
+                            {
+                                t.setShares(asShares(v.get("shares")));
+                            }
+                            t.setDate(asDate(v.get("date")));
+                        })
+
+                        .section("amount", "currency")
+                        .match("Steuerpflichtiger Betrag (.*) (?<currency>\\w{3}+) (?<amount>\\d{1,3}(\\.\\d{3})*(,\\d{2})?)")
+                        .assign((t, v) -> {
+                            t.setAmount(asAmount(v.get("amount")));
+                            t.setCurrencyCode(asCurrencyCode(v.get("currency")));
+                        })
+                        .wrap(t -> new TransactionItem(t));
+        addTaxesSectionsAccount(pdfTransaction);
+    }
+    
+    @SuppressWarnings("nls")
+    private void addTransferInTransaction()
+    {
+        DocumentType type = new DocumentType("Wir erhielten zu Gunsten Ihres Depots");
+        this.addDocumentTyp(type);
+
+        Block block = new Block("Wir erhielten zu Gunsten Ihres Depots(.*)");
+        type.addBlock(block);
+        
+        Transaction<BuySellEntry> pdfTransaction = new Transaction<BuySellEntry>();
+        pdfTransaction.subject(() -> {
+            BuySellEntry entry = new BuySellEntry();
+            entry.setType(PortfolioTransaction.Type.TRANSFER_IN);
+            return entry;
+        });
+        
+        block.set(pdfTransaction);
+        pdfTransaction
+                        .section("name", "isin")
+                        .find("Gattungsbezeichnung ISIN")
+                        .match("(?<name>.*) (?<isin>[^ ]\\S*)$")
+                        .assign((t, v) -> {
+                            t.setSecurity(getOrCreateSecurity(v));
+                        })
+                        
+                        .section("notation", "shares", "date")
+                        .find("Nominal Schlusstag Wert")
+                        //STK 28,000 02.12.2011 02.12.2011
+                        .match("(?<notation>^EUR|^STK) (?<shares>\\d{1,3}(\\.\\d{3})*(,\\d{3})?) (\\d+.\\d+.\\d{4}+) (?<date>\\d+.\\d+.\\d{4}+)(.*)")
```

Comment:
Gute Idee. Hab ich entsprechend abgeändert und auch gleich für den DKB Import nachgezogen (dort etwas angepasst weil in den DKB PDFs "Stück" statt STK steht...)